lib/vmdb/plugins.rb

```diff
@@ -40,6 +40,17 @@ def ansible_content
       end
     end
 
+    def ansible_runner_content
+      @ansible_runner_content ||= begin
+        map do |engine|
+          content_dir = engine.root.join("content", "ansible_tmp")
+          next unless File.exist?(content_dir.join("requirements.yml"))
+
+          [engine, content_dir]
```

Comment:
Would prefer if this returned a Hash like some of the other ones here.  Either that or create the wrapper struct like AutomateDomain or AnsibleContent.